eknrenderer/libeos-knowledge-renderer.map

```diff
@@ -0,0 +1,6 @@
+{
+    global:
```

Comment:
What does this do?

Reply:
I'm not entirely sure. I copied this from dmodel http://mesonbuild.com/Reference-manual.html#library: `link_depends` strings, files, or custom targets the link step depends on such as a symbol visibility map. The purpose is to automatically trigger a re-link (but not a re-compile) of the target when this file changes.

Reply:
It's the Meson pan-compiler-compatible equivalent of `--export-symbols-regex`. That is, symbols starting with `dm_` will be exported and all others will not. So `dm_` is not correct in the following line, it should be `eknr_`.

Reply:
Oh yikes, I must have missed this! I meant to put `eknr_` there but it must have slipped off my list. thanks!